src/components/shared/input/Input.tsx

```diff
@@ -0,0 +1,14 @@
+import "./Input.scss";
+
+export interface InputProps {
+  labelTitle: string;
+}
+
+export const Input = ({ labelTitle }: InputProps) => {
+  return (
+    <label className="label" htmlFor={labelTitle}>
```

Comment:
id should also be a separate props

Reply:
after discussion we agreed to leave it for now as it is.